ExampleApp/ExampleApp/Screens/Characteristics/ViewModel/CellViewModelItem/CharacteristicsViewModelItem.swift

```diff
@@ -0,0 +1,31 @@
+import RxBluetoothKit
+import UIKit
+
+class CharacteristicsViewModelItem: SectionModelItem {
+
+    var rowData: [Any] {
+        return characteristicsRowItems
+    }
+
+    var itemsCount: Int {
+        return characteristicsRowItems.count
+    }
+
+    var sectionName: String?
+
+    private(set) var characteristicsRowItems: [Characteristic]
+
+    init(_ sectionName: String, characteristicsRowItems: [Characteristic]?) {
```

Comment:
Not sure about wrapping the array into an observable. Is there a special meaning for the `nil` value? If not, maybe it's better to just ask for `[Characteristic]` and if you can see a client not having anything to pass, provide a default value `[]`?

Reply:
Done